Students/Hada Maria/Assignment_1/src/pack/SumOfMultiples.java

```diff
@@ -0,0 +1,52 @@
+package pack;
+
+import java.util.Scanner;
+
+/**
+ * 
+ * @author Maria
+ *
+ */
+public class SumOfMultiples {
+	/**
+	 * 
+	 */
+	private SumOfMultiples() {
+	}
+
+	/**
+	 * 
+	 * @param args
+	 *            -no param args
+	 */
+	public static void main(final String[] args) {
+
+		Scanner keyboardNumber = new Scanner(System.in);
+		System.out.print("Please enter a number: ");
+		int inputNumber = keyboardNumber.nextInt();
+		keyboardNumber.close();
+
+		int sumOfMultiples = 0;
+		final int firstNumber = 3;
+		final int secondNumber = 5;
+		final int thirdNumber = 15;
+		int mult3 = 0, mult5 = 0, mult15 = 0;
+
+		for (int i = 1; i < inputNumber; i++) {
```

Comment:
Check out [this post](http://math.stackexchange.com/questions/9259/find-the-sum-of-all-the-multiples-of-3-or-5-below-1000) to see how you could do this in constant time `O(1)` instead of liniar `O(n)`